### DISCUSSION TITLE
Build + release

### DISCUSSION THREAD

**Discussion by sergiy0** [joan38/kubernetes-client#101] [General]
Dear @joan38, 

Could you please take a look at the latest master build (https://github.com/joan38/kubernetes-client/runs/3616681008)? It has failed with `mill.scalalib.PublishModule.publishAll java.lang.RuntimeException: Failed to publish kubernetes-client_2.13, kubernetes-client_2.12 to Sonatype. Errors: Code: 401`

Thanks in advance,
Sergiy

**Comment by joan38** [comment]
Sorry I never replied but it's good now
